src/fidocadj/ADesktopIntegration.java

```diff
@@ -48,96 +49,107 @@ public class ADesktopIntegration implements AboutHandler, PreferencesHandler,
     }
 
     /** Check if some actions are made available by the operating system and if
-        it is the case, register them.
-        For example, MacOS usually inserts the About and Preferences menus in a
-        specific place in the "FidoCadJ" menu, that does not exist in other
-        platforms. This means that the OS needs to know what to call when one
-        of those two actions is selected. This is given to the operating system
-        by this routine.
-    */
+     * it is the case, register them.
+     * For example, MacOS usually inserts the About and Preferences menus in a
+     * specific place in the "FidoCadJ" menu, that does not exist in other
+     * platforms. This means that the OS needs to know what to call when one
+     * of those two actions is selected. This is given to the operating system
+     * by this routine.
+     */
     public void registerActions()
     {
-        if(!Desktop.isDesktopSupported()) {
+        if (!Desktop.isDesktopSupported()) {
             return;
         }
 
-        handleAbout=true;
-        handlePreferences=true;
-        Desktop d=Desktop.getDesktop();
+        handleAbout = true;
+        handlePreferences = true;
+        Desktop d = Desktop.getDesktop();
         try {
             d.setOpenFileHandler(this);
             d.setQuitHandler(this);
-        } catch(UnsupportedOperationException eE) {
+        } catch (UnsupportedOperationException eE) {
             // This can be ignored, we are going to live without it.
-            System.err.println(
-                "Warning: unsupported exception while setting handlers.");
+            /*
```

Comment:
There is a global variable called Globals.isBeta. It is supposed to be true for a preliminary version of FidoCadJ (i.e. a version that as a Greek letter after the numbers, such as 0.24.9 alpha). Such messages are useful when debugging but are probably never seen by the end user. You can modify this to show the warning on the console only if isBeta is true.